independent-publisher-connectors/NFTMania/NFTMania.md

```diff
@@ -0,0 +1,38 @@
+**Proposal for NFTMania Connector**
```

Comment:
@shreyan1999 please follow this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md) and for the raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md)

Reply:
I have added it in the reade.md with the mentioned template.